corehq/sql_db/__init__.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+from django.apps import AppConfig
+from django.conf import settings
+from django.core.checks import Error, register
+
+
+class SQLDBAppConfig(AppConfig):
+    name = 'corehq.sql_db'
+
+    def ready(self):
+        super(SQLDBAppConfig, self).ready()
+        register()(custom_db_checks)
```

Comment:
Is this line necessary? The function already has the `register` decorator and since this is in `__init__` it will get processed during import.

Reply:
Thanks, removed it!